Doc/faq/library.rst

```diff
@@ -111,7 +111,7 @@ Is there an equivalent to C's onexit() in Python?
 -------------------------------------------------
 
 The :mod:`atexit` module provides a register function that is similar to C's
-:c:func:`onexit`.
+:c:func:`!onexit`.
```

Comment:
@erlend-aasland This is the only instance of `onexit`, do you think it's worth adding this to `nitpick_ignore`?

https://github.com/python/cpython/blob/a47c13cae5b32e6f3d7532cc6dbb4e1ac31219de/Doc/conf.py#L79

Reply:
My view of ``nitpick_ignore`` is that we should aim to have it empty once this project is finished -- either by documenting currently undocumented things, or suppressing references -- hence suppressing here. But concious opinions may differ here.

A

Reply:
For only one instance, I'd just mark it up explicitly.